pkg/file/pipeline/bmt/bmt_test.go

```diff
@@ -29,9 +28,9 @@ func TestBmtWriter(t *testing.T) {
 		{
 			// this is a special case, since semantically it can be considered the hash
 			// of an empty file (since data is all zeros).
-			name:    "all zeros",
-			data:    make([]byte, swarm.ChunkSize),
-			expHash: mustDecodeString(t, "09ae927d0f3aaa37324df178928d3826820f3dd3388ce4aaebfc3af410bde23a"),
+			name:    "empty file",
+			data:    make([]byte, 0),
+			expHash: mustDecodeString(t, "b34ca8c22b9e982354f9c7f50b470d66db428d880c8a904d5fe4ec9713171526"),
```

Comment:
this hash is different than the current zero hash which is strange. also, it might be worth to change the comment of the test, which is now wrong

Reply:
it was incorrect what can i do :) ?